src/game/protos/scene.c

```diff
@@ -205,9 +205,9 @@ void scene_dynamic_tick(scene *scene, int paused) {
 }
 
 void scene_input_poll(scene *scene) {
-    if(scene->static_ticks_since_start < 25) {
+    if(scene->static_ticks_since_start < 25 && !(SCENE_ARENA0 <= scene->id && scene->id <= SCENE_ARENA4)) {
```

Comment:
There's a `scene_is_arena` function that would be more future-proof here, if we ever add more arenas.